piicatcher/dbinfo.py

```diff
@@ -13,7 +13,11 @@ def get_count_query(cls, schema_name: str, table_name: str) -> str:
 
     @classmethod
     def get_select_query(
-        cls, schema_name: str, table_name: str, column_list: List[str]
+        cls,
+        schema_name: str,
+        table_name: str,
+        column_list: List[str],
+        project_id: Optional[str] = None,
```

Comment:
The new Project ID parameter is specific to the BigQuery implementation. Doesn't seem right to include that in the abstract `DbInfo` base class. If other DB connectors also need their own specific parameters, this can get quite messy.

I'm not sure how these connector classes are being used. All the business logic seems to be in class methods so I assume the code never instantiate an instance of the connector class before using it? If the code does instantiate a new class instance, the class constructor would be a good way to pass the project ID, e.g. something like this:

```python
connector = BigQuery(project_id = "abc-123")
connector.get_select_query(cls, "schema", "table", ["column"])
```